### PR TITLE
style: simplify dependencies installed in CI

### LINKED ISSUE #69: `.github/workflows/test.yml` has duplicate dependencies

### PR DESCRIPTION
**Pull request by hamirmahal**
fixes #69.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,15 +9,8 @@ jobs:
...
-            librust-gstreamer-sys-dev \
             librust-gdk-sys-dev \
-            librust-atk-dev \
+            librust-gstreamer-sys-dev \
             librust-gtk4-sys-dev \
-            libgdk-pixbuf-2.0-dev \
-            librust-pango-dev \
-            librust-gobject-sys-dev \
-            ffmpeg
+            libunwind-dev
```
Comment (hamirmahal) [inline .github/workflows/test.yml:15]: This was missing in the commit that failed on the main branch.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -9,15 +9,8 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get install \
-            librust-gstreamer-sys-dev \
-            librust-gtk4-sys-dev \
-            libunwind-dev \
-            librust-gstreamer-sys-dev \
             librust-gdk-sys-dev \
```
Comment (hamirmahal) [inline .github/workflows/test.yml:12]: I believe
```yml
librust-gdk-sys-dev \
```
was missing in the commit that failed on the main branch.

Comment (hamirmahal) [reply .github/workflows/test.yml:12]: https://github.com/xlmnxp/blue-recorder/commit/93a4f02e81b2039bf2475258d05973de8465948e

**Review by me00001 — APPROVED** [review]
Great improvement, thanks!

**Comment by hamirmahal** [comment]
You're welcome!

**Pull request merged**